crates/matrix-sdk/src/room/timeline/event_item.rs

```diff
@@ -221,8 +217,6 @@ impl fmt::Debug for LocalEventTimelineItem {
             .field("sender", &self.sender)
             .field("timestamp", &self.timestamp)
             .field("content", &self.content)
```

Comment:
This `Debug` impl can be derived now. `sender_profile` being omitted before was not intentional.

Reply:
True!